dir.props

```diff
@@ -579,5 +579,6 @@
   </PropertyGroup>
 
   <!-- Use Roslyn Compilers to build -->
```

Comment:
What about the other properties you included in your BuildTools change? I would actually like to see them removed from this file as well.